inlong-agent/agent-common/src/main/java/org/apache/inlong/agent/constants/AgentConstants.java

```diff
@@ -160,4 +160,7 @@
 
     public static final String AGENT_LOCAL_IP = "agent.local.ip";
 
+    public static final String PROMETHEUS_EXPORTER_PORT = "agent.prometheus.exporter.port";
+    public static final int DEFAULT_PROMETHEUS_EXPORTER_PORT = 8080;
```

Comment:
It is recommended to use some infrequently used ports.

Reply:
@healchow the default port for Prometheus is 8080